<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliderPropertyGridWidget</class>
 <widget class="QWidget" name="SliderPropertyGridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>392</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>392</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="thumbSpriteLabel">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>110</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Thumb sprite:</string>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="minSpriteLabel">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>180</y>
      <width>68</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min sprite:</string>
    </property>
   </widget>
   <widget class="QLabel" name="drawTypeLabel">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>257</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min draw type:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="minDrawTypeComboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>253</y>
      <width>163</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="frameLabel">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>230</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Min frame:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="minFrameSpinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>227</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxFrameSpinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>327</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="drawTypeLabel_2">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>360</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max draw type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="frameLabel_2">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>331</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max frame:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="maxDrawTypeComboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>356</y>
      <width>163</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="frameLabel_3">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Thumb frame:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="thumbFrameSpinBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>157</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="removeMaxSpriteButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxSpriteLineEdit">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>300</y>
      <width>214</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="maxSpriteLabel">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>280</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max sprite:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openMaxSpriteButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minSpriteLineEdit">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>200</y>
      <width>214</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="removeMinSpriteButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openMinSpriteButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="thumbSpriteLineEdit">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>130</y>
      <width>214</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="removeThumbSpriteButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openThumbSpriteButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="maxValueSpin">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>87</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="maxValueLabel">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="minValueLabel">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>61</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Value:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="minValueSpin">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>58</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="valueSpin">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QSlider" name="valuePosition">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>29</y>
      <width>200</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
    <property name="tickInterval">
     <number>5</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
